manifests/init.pp

```diff
@@ -235,6 +240,31 @@
     order   => '01',
   }
 
+  # Create any DHCP classes requested
+  if $dhcp_classes {
```

Comment:
If you force it to be a hash, then this check is redundant since ```create_resources``` can handle an empty hash.